docker-compose.yml

```diff
@@ -42,6 +42,9 @@ services:
       - POSTGRES_HOST=database
       - POSTGRES_PORT=5432
       - KHOJ_DJANGO_SECRET_KEY=secret
+      # uncomment KHOJ_NO_HTTPS and KHOJ_DOMAIN with your local domain and comment out KHOJ_DEBUG if running on LAN without SSL (CSRF verification failed)
+      # - KHOJ_NO_HTTPS=True
+      # - KHOJ_DOMAIN=khoj.home.local # replace with your own domain     
```

Comment:
You can also set KHOJ_DOMAIN to the I.P of the machine Khoj is running on. It'd be great if we can update the comment to indicate this

```suggestion
      # - KHOJ_DOMAIN=khoj.home.local # replace with your own domain or I.P
```